march_hardware_builder/include/march_hardware_builder/hardware_builder.h

```diff
@@ -93,4 +93,13 @@ class HardwareBuilder
   bool init_urdf_ = true;
 };
 
+/*
```

Comment:
docstrings should start with `/**` :innocent: 
```suggestion
/**
```